src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-anchoredit.js

```diff
@@ -98,6 +107,16 @@ export default class EzBtnAnchorEdit extends Component {
         });
     }
 
+    isValueValid(value) {
+        const pattern = /^[A-Za-z][A-Za-z0-9\-_:\.]*$/;
```

Comment:
I think it would be good to move it to the top of the file and name it `ANCHOR_PATTERN` or something like that